scripts/zones/Sealions_Den/mobs/Omega.lua

```diff
@@ -15,21 +15,22 @@ end
 
 entity.onMobSpawn = function(mob)
     mob:setBehaviour(bit.bor(mob:getBehaviour(), xi.behavior.NO_TURN))
+    mob:setMobMod(xi.mobMod.SKILL_LIST, 54)
 end
 
 entity.onMobFight = function(mob)
     -- Gains regain at under 25% HP
-    if
-        mob:getHPP() < 25 and not
-        mob:hasStatusEffect(xi.effect.REGAIN)
-    then
-        mob:addStatusEffect(xi.effect.REGAIN, 5, 3, 0)
-        mob:getStatusEffect(xi.effect.REGAIN):setFlag(xi.effectFlag.DEATH)
-    -- Changes AA round delay as it gets weaker
-    elseif mob:getHPP() < 60 then
+    local stage = mob:getLocalVar("stage")
+
+    if mob:getHPP() < 60 and stage == 0 then
         mob:setDelay(3000)
-    elseif mob:getHPP() < 25 then
+        mob:setLocalVar("stage", 1)
+    elseif mob:getHPP() < 25 and stage < 2 then
         mob:setDelay(2500)
+        mob:addStatusEffect(xi.effect.REGAIN, 5, 3, 0)
+        mob:getStatusEffect(xi.effect.REGAIN):setFlag(xi.effectFlag.DEATH)
```

Comment:
I don't think retail uses the actual status effect for this does it? Can a thf nab it with aurasteal procs? some pretty crazy things can be aura-stole including the darkrider's invincibility (which immediately wears off and self-reapplies to him, but it's still pretty funny to witness) but I'd wager what we probably need ot do is setMod the regain modifier to turn this on and off.

Reply:
Honestly not sure and I can change it if desired. My guess is that it is likely a modifier. I was just relocating that chunk of code to this section. On my backlog I have it to go through all of the CoP battlefields and fix them up to have proper mechanics.

Reply:
+1 to move to a setMod instead of a status effect, would be pretty cool if retail could let you yoink a regain effect though!

Reply:
Ok, I changed it to be a mod and to increase over time which is what Eden has and is based off of this awesome capture by Jimmayus: https://discord.com/channels/443544205206355968/443552700064333835/644089353278193665